From a7ee1fcdae89c17721a12240d4d6b775820b60ed Mon Sep 17 00:00:00 2001
From: Sanskriti Sharma <sansharm@redhat.com>
Date: Tue, 9 Oct 2018 15:08:12 -0400
Subject: [PATCH 13/25] numactl: add va_end to usage function

Fixes the following cppcheck:

  [numactl.c:104]: (error) va_list 'ap' was opened but not closed by va_end().

Signed-off-by: Sanskriti Sharma <sansharm@redhat.com>
---
 numactl.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/numactl.c b/numactl.c
index a2b2d9d..76a1cca 100644
--- a/numactl.c
+++ b/numactl.c
@@ -101,6 +101,7 @@ void usage_msg(char *msg, ...)
 	vfprintf(stderr, msg, ap);
 	putchar('\n');
 	usage();
+	va_end(ap);
 }
 
 void show_physcpubind(void)
-- 
1.8.3.1

